cli/cmd/install_test.go

```diff
@@ -17,7 +17,7 @@ func TestRender(t *testing.T) {
 		t.Fatalf("Unexpected error: %v", err)
 	}
 
-	defaultValues, _, err := defaultOptions.validateAndBuild("", nil)
+	defaultValues, _, err := defaultOptions.validateAndBuild("", defaultOptions.recordableFlagSet())
```

Comment:
Why'd all of these switch from `nil` to taking the flag set?

Reply:
IIRC that was part of @cpretzer's patch and I am somewhat curious myself. :)